report/5.Conclusion/Discussion.tex

```diff
@@ -1,4 +1,62 @@
-% !TeX root = ../main.tex
-\chapter{Discussion}
-
+% !TeX root = ../main.tex

+\chapter{Discussion}
+\section{Discussion}
+In this chapter, the project is reflected upon.

```

Comment:
Not sure, but didn't Ingo complain about this kind of meta comment?

Reply:
His problem was more with the tense. It is good to describe what is going on in a chapter.
Still, I will be changing it a bit, mostly because there is some redundancy.